public/app/plugins/datasource/tempo/traceql/autocomplete.ts

```diff
@@ -461,23 +445,20 @@ export class CompletionProvider implements monacoTypes.languages.CompletionItemP
  * @param type
  * @param monaco
  */
-function getMonacoCompletionItemKind(
-  type: CompletionItemType,
-  monaco: Monaco
```

Comment:
we don't need this just to pull out an Enum, but grafana-ui's `monacoTypes` export is only `types`, so cannot be used as actual values, so i had to import the enum exported by monaco lib itself :neutral_face: